graph-db/jest.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  clearMocks: true,
+  testEnvironment: 'node'
```

Comment:
I would add the following to your config so that you can see your overall test coverage which will help keep your eye on the EED goal:

``` 
  collectCoverage: true,
  collectCoverageFrom: [
    'src/**/*.js'
  ]
```